<launch>

	<include file="$(find stereo_blockmatcher)/launch/real_time_pipeline.launch" />
	<node pkg="cadrl_ros" type="pedestrian_merger_node.py" name="pedestrian_merger_node"/>
	

	<arg name="jackal_speed" default="0.5"/>
    <!-- Launch neural net ros wrapper -->
    <node pkg="cadrl_ros" type="cadrl_node.py" name="cadrl_node" output="screen" ns="/JA01">

        <!-- Publications -->
        <remap from="~other_vels" to="other_vels"/>
        <remap from="~nn_cmd_vel" to="nn_cmd_vel"/>
        <remap from="~pose_marker" to="pose_marker"/>
        <remap from="~path_marker" to="path_marker"/>
        <remap from="~goal_path_marker" to="goal_path_marker"/>
        <remap from="~agent_marker" to="other_agents_marker"/>
        <remap from="~agent_markers" to="other_agents_markers"/>

        <!-- Subscriptions -->
        <remap from="~pose" to="/tf"/>
        <remap from="~velocity" to="velocity"/>
        <remap from="~safe_actions" to="local_path_finder/safe_actions"/>
        <remap from="~planner_mode" to="planner_fsm/mode"/>
        <remap from="~goal" to="/move_base_simple/goal"/>
				<remap from="~wheel_odo" to="/pioneer_interface/pose"/>

        <!-- Parameters -->
        <param name="~jackal_speed" value="$(arg jackal_speed)"/>

    </node>

	<!--node type="rviz" name="rviz" pkg="rviz"/-->
</launch>
